WordPress/Classes/ViewRelated/Gutenberg/GutenbergViewController.swift

```diff
@@ -823,11 +823,16 @@ extension GutenbergViewController: GutenbergBridgeDelegate {
 
     func gutenbergDidMount(unsupportedBlockNames: [String]) {
         if !editorSession.started {
+            var capabilities: [String: Bool] = [:]
+            for ability in self.gutenbergCapabilities() {
```

Comment:
nitpick: I think you could use something like `for (key, value) in gutenbergCapabilities()` here.

Reply:
Love it! Give me all the nitpicks you have, because I'm not as familiar with Swift. 😄